src/edgeworkers/ew-service.ts

```diff
@@ -1,19 +1,11 @@
 import * as envUtils from '../utils/env-utils';
 import * as cliUtils from '../utils/cli-utils';
+import * as httpEdge from '../cli-httpRequest'
 import * as fs from 'fs';
 
 var accountKey: string = null;
-
 const EDGEWORKERS_API_BASE = '/edgeworkers/v1';
 
-export function setAccountKey(account: string) {
```

Comment:
I think you moved this account switch key logic into httpRequest which is fine, but please make sure to test that it still works as Professional Services uses this alot to manage multiple customers.

Reply:
@Ehsap  Justin do we have regression for this ?

Reply:
Yes, we have regression tests that cover the `--accountkey` argument scenario.